ui/ozone/platform/starboard/gl_ozone_egl_starboard.cc

```diff
@@ -51,10 +261,35 @@ bool GLOzoneEGLStarboard::LoadGLES2Bindings(
     const gl::GLImplementationParts& implementation) {
   DCHECK_EQ(implementation.gl, gl::kGLImplementationEGLANGLE)
       << "Not supported: " << implementation.ToString();
-  // TODO(b/371272304): call into LoadDefaultEGLGLES2Bindings instead and let
-  // Angle load GLES and EGL.
-  gl::GLGetProcAddressProc gl_proc = reinterpret_cast<gl::GLGetProcAddressProc>(
-      SbGetEglInterface()->eglGetProcAddress);
+  gl::GLGetProcAddressProc gl_proc =
+      [](const char* name) -> gl::GLFunctionPointerType {
+    gl::GLFunctionPointerType proc =
+        reinterpret_cast<gl::GLFunctionPointerType>(
+            SbGetEglInterface()->eglGetProcAddress(name));
+    if (proc) {
+      return proc;
+    }
+    // For EGL 1.4, eglGetProcAddress might not return pointers to EGL core
+    // functions. In this case, we need to retrieve them from the
+    // SbEglInterface.
```

Comment:
Is this so? eglGetProcAddress was introduced in eGL 1.0, why would
it fail for eGL 1.4? IOW this might be a specific Raspi implementation
issue. A quick Google search seems to say that this is in fact 
true (https://screenshot.googleplex.com/Btvf7eK8yhVE2vk) and that
calling `eglGetProcAddress()` on Raspbian, when using dlsym (our
case I think?) seems to be broken (my search was "eglGetProcAddress 
raspbian bug").